src/FsAutoComplete.Core/FCSPatches.fs

```diff
@@ -276,9 +276,146 @@ type FSharpParseFileResults with
 
             override _.VisitBinding(defaultTraverse, binding) =
                 match binding with
-                | SynBinding.Binding (_, _, _, _, _, _, _, _, _, expr, _range, _) as b when rangeContainsPos b.RangeOfBindingAndRhs pos ->
+                | SynBinding.Binding (_, _, _, _, _, _, _, _, _, expr, _range, _) as b when Range.rangeContainsPos b.RangeOfBindingAndRhs pos ->
                     tryGetIdentRangeFromBinding b
                     |> Option.bind (walkBinding expr)
                 | _ -> defaultTraverse binding
         })
     | None -> None
+
+
+module SyntaxTreeOps =
+  open FSharp.Compiler.SyntaxTree
+  let rec synExprContainsError inpExpr =
```

Comment:
I had to add this directly from the dotnet/fsharp source because it was no longer available, @cartermp can we look at adding it back in, or do you know of a replacement member for `synExprContainsError`?